server/tests/helpers.py

```diff
@@ -78,9 +78,11 @@ def create_user(email=DEFAULT_AA_EMAIL) -> User:
 
 
 def create_org_and_admin(
-    org_name: str = "Test Org", user_email: str = DEFAULT_AA_EMAIL
+    org_name: str = None, user_email: str = DEFAULT_AA_EMAIL
```

Comment:
nit: are we sure we want the API contract for this call to be that if you explicitly pass in a `None`, then it gets replaced with `Test Org` ? Is that sufficiently clear?

Reply:
the way this test helper usually gets used is with no arguments passed in because the test case doesn't care about the org name or user email. in the rare test cases that do care, they can configure them using these optional arguments. not totally sure if that answers your question though.